core/src/main/scala/akka/persistence/r2dbc/cleanup/scaladsl/EventSourcedCleanup.scala

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright (C) 2022 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.persistence.r2dbc.cleanup.scaladsl
+
+import scala.collection.immutable
+import scala.concurrent.Future
+import scala.util.Failure
+import scala.util.Success
+
+import akka.Done
+import akka.actor.ClassicActorSystemProvider
+import akka.actor.typed.ActorSystem
+import akka.actor.typed.scaladsl.LoggerOps
+import akka.annotation.ApiMayChange
+import akka.annotation.InternalApi
+import akka.persistence.SnapshotSelectionCriteria
+import akka.persistence.r2dbc.ConnectionFactoryProvider
+import akka.persistence.r2dbc.R2dbcSettings
+import akka.persistence.r2dbc.journal.JournalDao
+import akka.persistence.r2dbc.snapshot.SnapshotDao
+import org.slf4j.LoggerFactory
+
+/**
+ * Scala API: Tool for deleting all events and/or snapshots for a given list of `persistenceIds` without using
+ * persistent actors. It's important that the actors with corresponding `persistenceId` are not running at the same time
+ * as using the tool.
+ *
+ * WARNING: deleting events is generally discouraged in event sourced systems.
+ *
+ * If `neverUsePersistenceIdAgain` is `true` then the highest used sequence number is deleted and the `persistenceId`
+ * should not be used again, since it would be confusing to reuse the same sequence numbers for new events.
```

Comment:
`neverUse` might indicate that we will somehow block/lock this `persistenceId`, but this is actually the user's responsibility to not use this persistence id again. Actually, in some cases, it might be perfectly fine to use it again and start seq num from the begging. How about renaming it to `resetSequenceNumber`?

Reply:
sounds like a good suggestion